### PR TITLE
Update plots placeholders

### PR DESCRIPTION
**Pull request by SusannaV**

### CODE REVIEW

**Review by nicou — APPROVED** [review]
Good stuff 👍

File: src/components/Plot.js [hunk 1]
```diff
@@ -1,115 +1,221 @@
...
+              </Row>
+              {plot.events.length<1 ? <p>No linked events</p> : <ul> {plot.events.map(e => <li key={e.id}>
+                  <Link onClick={() => props.changeTab('Events')} to={`/events/${e.id}`}>{e.name}</Link></li>)}
+                  </ul>
+              }
+              <Row>
+                <Col sm><span className='mini-header'>Messages <span className='new'>[CREATE MESSAGE BUTTON]</span></span></Col>
+              </Row>
+              {relatedMessages.length<1 ? <p>No messages</p> : <ul> {relatedMessages.map(m => <li key={m.id}>
+                  <Link onClick={() => props.changeTab('Messages')} to={`/messages/${m.id}`}>{m.name}</Link> {m.sent ? "[Already sent]" : "[Not send yet"}</li>)}
```
Comment (5annaha) [inline src/components/Plot.js:133]: m.sent is not boolean, it is a string. Now it always returns true. This can be only [{m.sent}].

**Review by 5annaha — CHANGES_REQUESTED** [review]
Looks good to me. Just fix Messages m.sent and it is good to go.

File: src/components/Plot.js (hunk 1 above)
Comment (SusannaV) [reply src/components/Plot.js]: An excellent catch! Thank you :) Fixed it. Went with "Sent: {m.sent}" for clarity.

**Review by 5annaha — APPROVED** [review]
lgtm

**Pull request merged**